extensions-core/kafka-indexing-service/src/main/java/org/apache/druid/indexing/kafka/IncrementalPublishingKafkaIndexTaskRunner.java

```diff
@@ -84,7 +84,7 @@ public IncrementalPublishingKafkaIndexTaskRunner(
   }
 
   @Override
-  protected Long getSequenceNumberToStoreAfterRead(@NotNull Long sequenceNumber)
+  protected Long getNextStartOffset(@NotNull Long sequenceNumber)
```

Comment:
:+1 on switching to 'offset', i think it's more intuitive terminology, though maybe change parameter variable name too?

Reply:
I decided to revert this for now, but plan to try again later.